### ISSUE TITLE
Form Validation instance clashing with other Form validation instances

### ISSUE THREAD

**Issue by razorsharpshady** [yidas/codeigniter-model#9]
Hi Nick,
The **validate** method in Model.php has an issue in 

```
$this->load->library('form_validation');
        $this->form_validation->set_data($data);

        $this->form_validation->set_rules($rules);
      // Run Validate
        $result = $this->form_validation->run();
```
If there is other **form validation** instance present which is accessible $this->form_validation then  **validate** method applies rules from other instances as well(because it's the same instance infact). 
I think you should change the instance name from form_validation to something random and unique to a model. 
p.s Great work.

**Comment by yidas** [comment]
Hi @razorsharpshady,

Very good point!

This issue has been fixed at [version 2.13.1](https://github.com/yidas/codeigniter-model/releases/tag/2.13.1)

Thank you!
Nick

**Comment by razorsharpshady** [comment]
Awesome!!
